### ISSUE TITLE
@snippetdoc didn't tansform markdown table

### ISSUE THREAD

**Issue by zhangzq** [doxygen/doxygen#6483]
if the snippet is 

```
/// [test_snippet]
| h1 | h2
|--|--
|v1 | v2
/// [test_snippet]
```

then `@snippetdoc snippet.md test_snippet ` will not tansform the table in markdown, it just shows the snippet without line breaks. 

It works find when I wrote the table in html, i.ie `<table> ,,, </table>`。But it's too urgly and hard to modify.

**Comment by zhangzq** [comment]
I found that neither @includedoc can tansform a markdown table.

**Comment by albert-github** [comment]
This looks like one of the exceptions, but not documented, in the category:

> Some commands like \cond and \if don't work with this command due to the moment of parsing.
> The included documentation should not have comment signs in it as they will appear in the documentation as
> well.

Note also that a similar problem was mentioned in https://stackoverflow.com/questions/52120027/doxygen-how-to-include-a-markdown-page-to-document-a-group

**Comment by zhangzq** [comment]
@albert-github I found you flag this as a bug. Will it be fixed later? May I ask your schedule?

It will help a lot in dealing with repeated documents.

**Comment by albert-github** [comment]
It might be fixed (depending on the possibility to fix it) but there is no schedule for it (yet).

**Comment by albert-github** [comment]
It looks like this problem is fixed in the current master (most likely by means of the pull request #8395).
Please verify, but please don't close the issue when it works now as this will be, automatically, done at the moment of an official release.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.2.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
